ui/raidboss/data/06-ew/trial/hydaelyn-ex.txt

```diff
@@ -1,25 +1,31 @@
-# Hydaelyn Extreme
+### Hydaelyn Extreme
+# -p 65BC:336.2 65BB:1433.6
+# -ii 65A7 6F12 65BA 6507 65AA 6596 6690 65BE 686C 6C92
```

Comment:
@Legends0 This is what I meant about `-ii` stuff.  You can pass `-ii` parameters to ignore specific abilities when calling `make_timeline` rather than just manually trimming them.  I've started listing out what abilities those are when making timelines so that others can easily reproduce the timeline in the future.  `-p` is to add a phase sync to particular time for a particular ability. `-it` will give you targetable/untargetable lines for particular mob names too.